lsst-epo/investigations-client
# [F] add XLSX export

For JIRA: "EPO-8809 #IN-REVIEW #comment add XLSX export"

JIRA: https://jira.lsstcorp.org/browse/EPO-8809

## What this change does ##

Add XLSX export button to review page that makes a Next server API call and builds an XLSX output of the users answers.

## Notes for reviewers ##

Client-side change, make sure to `yarn` to get the latest widget library.

Include an indication of how detailed a review you want on a 1-10 scale.
- 5 = "Please make sure there are no obvious errors and that you believe it does what it says it does"

## Testing ##

- do some answers, make sure to include a Color Tool widget
- go to the review page, click the "Export" button
- view the XLSX file that has downloaded and see that it contains the text and images matching your work.

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##
### Before:
### After:
